pkg/plugins/pg/README.md

```diff
@@ -1,144 +1,149 @@
 Conduit PostgreSQL Connectors 
-============================
+=============================
```

Comment:
Nitpick: we should stick to one way of defining headers, probably using `#`.

Reply:
I do this when it's the entire document header, to distinguish it from other level 1 headers in the doc, but noted.